### DISCUSSION TITLE
设置问题：新手求教！

### DISCUSSION THREAD

**Discussion by ghost** [community/community#29805] [New to GitHub]
在我的仪表盘中，能看到别人的库，我该怎么才能隐藏别人的库！只显示自己的库呢？

**Comment by queenofcorgis** [comment]
Hi there - you are able to filter out what you see in "For You" on the Feed if that helps! 
![Screen Shot 2022-08-17 at 1 22 54 PM](https://user-images.githubusercontent.com/92824518/185202911-5619791a-52bd-4669-8f7a-c3108d3b76f0.png)
